### PR TITLE
H-2588: Increase ELB timeout and interval

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

The migration script may takes a while to complete

### 🚢 Has this modified a publishable library?

This PR:

- [x] does not modify any publishable blocks or libraries, or modifications do not need publishing

### 📜 Does this require a change to the docs?

The changes in this PR:

- [x] are internal and do not require a docs change

### 🕸️ Does this require a change to the Turbo Graph?

The changes in this PR:

- [x] do not affect the execution graph

## ⚠️ Known issues

This is a temporary solution. The migration should run in a container which expects to exit, then the health-checked service should be started

### CODE REVIEW

**Review by CiaranMn — COMMENTED** [review]
These values apply to all health checks for the entire lifetime of the service, which means it'll be much slower for services that become unhealthy to be detected and restarted / alerted – I think we're better off amending `app_grace_period_seconds`, which is the time period after the service is initialized during which health checks won't count as failures. `300` should do it, maybe `500` to be safe.

This will set the grace period for both the load balancer, and the container's own health checks.

**Comment by TimDiekmann** [comment]
> These values apply to all health checks for the entire lifetime of the service, which means it'll be much slower for services that become unhealthy to be detected and restarted / alerted – I think we're better off amending `app_grace_period_seconds`, which is the time period after the service is initialized during which health checks won't count as failures. `300` should do it, maybe `500` to be safe.
> 
> This will set the grace period for both the load balancer, and the container's own health checks.

I didn't know that `app_grace_period_seconds` also affects the ELB, changed the grace periods to 500. Thank you!

File: infra/terraform/hash/hash_application/main.tf [hunk 1]
```diff
@@ -149,7 +149,7 @@ resource "aws_lb" "app_alb" {
 
   security_groups = [aws_security_group.alb_sg.id]
   # Timeout is set to allow collab to use long polling and not closing after default 60 seconds.
-  idle_timeout    = 4000
+  idle_timeout = 4000
 }
```
[the remark this answers is not in the document]
Comment (TimDiekmann) [reply infra/terraform/hash/hash_application/main.tf]: @CiaranMn we probably want to fix this I guess?

**Comment by TimDiekmann** [comment]
I set the value to `300` as this is the maximum allowed value.

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**